_posts/blog/2021-01-18-Open-position-Cryptography-Engineer.md

```diff
@@ -0,0 +1,90 @@
+---
+layout: post
+title: "Open Position: Cryptography Engineer"
+excerpt: "You can find application details below."
+categories: blog
+tags: [staff]
+date: 2021-01-18
+author: antonie
+---
+
+
+The Zcash Foundation is a 501(c)(3) nonprofit, public charity that builds and supports privacy infrastructure for the public good. We work on strengthening financial privacy with technology, focused on the Zcash protocol and blockchain.
+
+We're looking for someone who is as excited as we are about building private financial infrastructure for the public good, and we take that task very seriously.
+
+* Why work for the Zcash Foundation?
+* We are a small start-up nonprofit that punches way above its weight.
+* Everything we build is open source, and we develop in the open.
+* Our mission and values are very important. We want to work with like-minded individuals who share them with us.
+* We are fully remote and have team members all over the globe.
+* Leadership cares deeply about your health and fulfillment.
+* We offer comprehensive benefits that include full coverage health insurance for employees and dependents, retirement savings account, parental leave, Bring your own Device (BYOD) annual benefit, and a generous Paid Time Off policy.
+* Nonprofit means what it says! We build what serves our mission.
+# Role: Cryptography Engineer
+The role as a cryptography engineer within the core Zcash Foundation team will be responsible for building cryptographic protocols as well as distributed systems.  The ideal candidate embodies the Foundation's [values](https://www.zfnd.org/about/#values), while fully aligning with its [mission](https://www.zfnd.org/about/#mission) and [goals](https://www.zfnd.org/about/#goals). 
+
+Engineers at the Zcash Foundation are responsible for implementing the core Zcash protocol, maintaining deployed software, fixing bugs, and identifying improvements to the protocol for the future. Other duties include writing about our work and interfacing with external stakeholders such as those who use our software and interoperable implementations of the Zcash protocol. The position reports to the Zcash Foundation's engineering manager.
+
+Zcash Foundation Core Engineering Projects: Currently the engineering team is working on [Zebra](https://github.com/ZcashFoundation/zebra), an independent implementation of the Zcash protocol written in Rust, and soon we will dedicate resources on building out Zcash wallet functionality.
```

Comment:
I don't know if "on vs to" is a dialect thing - either way the meaning is clear.

```suggestion
Zcash Foundation Core Engineering Projects: Currently the engineering team is working on [Zebra](https://github.com/ZcashFoundation/zebra), an independent implementation of the Zcash protocol written in Rust, and soon we will dedicate resources to building out Zcash wallet functionality.
```